packages/client/ui/react-ui/src/components/embed/CrossmintPaymentElement.test.tsx

```diff
@@ -17,13 +17,13 @@ describe("CrossmintPaymentElement", () => {
         expect(iframe).toHaveAttribute("src");
         expect(iframe).toHaveAttribute("id", "crossmint-embedded-checkout.iframe");
         expect(iframe).toHaveStyle({
-            padding: "0px !important",
```

Comment:
multiple sources online site that passing `!important` through the style prop does not work, which is why this is not working...